indexer/src/types/transaction.ts

```diff
@@ -270,20 +356,24 @@ function fromSerializedData(bytes: Uint8Array): TypedTransaction {
     // will check if v >= 37. Since we pass it [v, r, s] = [chain_id, 0, 0], we need to force
     // the chain id to be >= 37. This value will be updated during the call to addSignature.
     values[6] = bigIntToBytes(37n);
-    return LegacyTransaction.fromValuesArray(
+
+    const typedTransaction = LegacyTransaction.fromValuesArray(
       values as TxValuesArray[TransactionType.Legacy],
-    );
+    )
+    
+    return typedTransaction;
```

Comment:
Why do we need to change that? I liked the previous compact format more

Reply:
changed back to the original code